### ISSUE TITLE
Accessibility enhancement: distinct aria-labels for each School Marker

### ISSUE THREAD

**Issue by trushmi** [sfbrigade/support-sfusd#252]
### What is the issue you are reporting?

**Description:** 

The aria-label attribute for the map markers is currently set to the same value for each school on the map.

To enhance accessibility for screen readers, each marker should have a unique aria-label to help users differentiate between schools.

Solution example: aria-label="School marker: Lincoln High School”.
